client/coniksclient/internal/cmd/run.go

```diff
@@ -0,0 +1,198 @@
+package cmd
+
+import (
+	"log"
+	"net/url"
+	"os"
+	"strings"
+
+	"github.com/coniks-sys/coniks-go/client"
+	"github.com/coniks-sys/coniks-go/keyserver/testutil"
+	p "github.com/coniks-sys/coniks-go/protocol"
+	"github.com/spf13/cobra"
+	"golang.org/x/crypto/ssh/terminal"
+)
+
+const help = "\n- register [name] [key]:\r\n" +
```

Comment:
> I though an extra newline looked nicer

Then we should add newline for all commands. Just a minor point. If you think it's good, we should do it.